src/App.tsx

```diff
@@ -1,13 +1,29 @@
 import React from 'react';
+import {
+  Switch,
+  Route,
+} from "react-router-dom";
 import Home from './components/Home';
-
+import Settings from './components/Settings';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import Login from './components/Login';
+import Article from './components/Article';
 
 
 function App() {
   return (
+    <main><Header />
```

Comment:
I would also recommend that you move the `<Header>` and `<Footer>` outside the `<main>` tag since that HTML tag is supposed to denote where on the page if your main section of content.

Reply:
For now the main tag is out, should return later in the page components